.github/workflows/sel4test-sim.yml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021, Proofcraft Pty Ltd
+#
+# SPDX-License-Identifier: BSD-2-Clause
+
+# sel4test simulation runs
+#
+# See sel4test-sim/builds.yml in the repo seL4/ci-actions for configs.
+
+name: seL4Test-Sim
+
+on:
+  workflow_call:
+
+jobs:
+  sim:
```

Comment:
It seem the "test with" in PRs works well for "seL4test", "seL4bench" and "sim". But I'm not fully understanding when we have to do a code freeze step an when not. Seems we have 3 different way how things are done for seL4test, seL4bench and sim. So a follow up task once this is merged should be aligning this better.

Reply:
`Test with` works with all of them. It's part of the checkout scripts they all call. Code freeze just eliminates a race condition where one test job might see a different manifest to another one (so we get the one manifest in the beginning and give it to all jobs).